### PR TITLE
CGE-109: Change RegEx in password and username fields according to checklist

### PR DESCRIPTION
**Pull request by KozminAlexandr**
Add permissible symbols to RegEx in password and username fields

### CODE REVIEW

**Review by lubchik-14 — CHANGES_REQUESTED** [review]
Unfortunately my decision to approve was made too quick. Your regexps don't work 😢

**Review by rpkyrych — APPROVED** [review]
Looks great!!! :+1: :1st_place_medal:

**Review by Jack11M — APPROVED** [review]

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -6,7 +6,8 @@ invalid.max-size=${validatedValue} must be less than {max} symbols
 invalid.email=Please enter valid email address (Ex: username@example.com)
 invalid.username=Please enter valid login, do not use space
 invalid.past-or-present.date=${validatedValue} date couldn't be in the future
-invalid.password=Your password must be between 8 and 30 characters at must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
+invalid.password=Your password must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
+invalid.password.size=Your password must be between 8 and 30 characters long
```
Comment (lubchik-14) [inline src/main/resources/messages.properties:10]: Could you please add the same message changes to test/resources/messages.properties

Comment (KozminAlexandr) [reply src/main/resources/messages.properties:10]: Thank you, very attentive remark =) Done.

**Review by lubchik-14 — APPROVED** [review]

**Pull request merged**